src/patternfly/components/Sidebar/sidebar.scss

```diff
@@ -148,6 +157,20 @@ $pf-c-sidebar__panel--list--width: ("default", 25, 33, 50, 66, 75, 100);
     --pf-c-sidebar__panel--Position: static;
     --pf-c-sidebar__panel--Top: auto;
   }
+
+  > .pf-c-tabs {
+    --pf-c-tabs--m-vertical--MaxWidth: auto; // when tabs are part of the sidebar, unset max width
+
+    &.pf-m-vertical.pf-m-box {
+      --pf-c-tabs__link--before--BorderRightColor: transparent;
+      --pf-c-tabs--m-vertical__list--before--BorderColor: transparent;
```

Comment:
These borders really don't make sense when the tabs are expandable. I wonder if we could find out how people are using the expandable tabs list currently? I imagine the tabs shouldn't have a width constraint or the borders or anything, then we wouldn't have to recreate the borders here.